sync-online/service-worker.js

```diff
@@ -0,0 +1,145 @@
+// [Working example](/serviceworker-cookbook/sync-online/).
+
+var CACHE_NAME = 'dependencies-cache';
+
+// Files required to make this app work offline
+var REQUIRED_FILES = [
+  '/',
```

Comment:
Since I'm using a scope of '.' for the SW registration, shouldn't '/' == '/sync-online/'